src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -1,249 +1,194 @@
+                            [invalidText]="passwordError"
+                            i18n>Password
+        <cd-helper *ngIf="passwordPolicyHelpText.length > 0"
+                    class="text-pre-wrap"
+                    html="{{ passwordPolicyHelpText }}">
+        </cd-helper>
+        <input cdsPassword
+               type="password"
+               placeholder="Password..."
+               id="password"
+               name="password"
+               autocomplete="new-password"
+               formControlName="password"
+               >
+        </cds-password-label>
+        <ng-template #passwordError>
+          <span class="invalid-feedback"
+                *ngIf="userForm.showError('password', formDir, 'match')"
+                i18n>Password confirmation doesn't match the password.
+          </span>
+          <span class="invalid-feedback"
+                *ngIf="userForm.showError('password', formDir, 'required')"
+                i18n>This field is required.</span>
+          <span class="invalid-feedback"
+                *ngIf="userForm.showError('password', formDir, 'passwordPolicy')">
+            {{ passwordValuation }}
+          </span>
+        </ng-template>
+      </div>
 
+      <!-- Confirm password -->
+      <div class="form-item">
+        <cds-password-label for="confirmpassword"
+                            label="Confirm password..."
+                            [invalid]="!userForm.controls.confirmpassword.valid && userForm.controls.confirmpassword.dirty"
+                            [invalidText]="confirmpasswordError"
+                            i18n> Confirm password
+          <input cdsPassword
+                 type="password"
+                 placeholder="Confirm password..."
+                 id="confirmpassword"
+                 name="confirmpassword"
+                 formControlName="confirmpassword">
+        </cds-password-label>
+        <ng-template #confirmpasswordError>
+          <span class="invalid-feedback"
+                *ngIf="userForm.showError('confirmpassword', formDir, 'match')"
+                i18n>Password confirmation doesn't match the password.</span>
+          <span class="invalid-feedback"
+                *ngIf="userForm.showError('confirmpassword', formDir, 'required')"
+                i18n>This field is required.</span>
+        </ng-template>
+      </div>
+      <!-- Password expiration date -->
+      <div class="form-item"
+           *ngIf="!authStorageService.isSSO()">
+        <cd-date-time-picker [ngClass]="{'required': pwdExpirationSettings.pwdExpirationSpan > 0}"
+                             [control]="userForm.get('pwdExpirationDate')"
+                             name="Password expiration date"
+                             placeholder="Password expiration date"
+                             [hasTime]="false"
+                             [defaultDate]="action == 'Create'? true : false"
```

Comment:
```suggestion
                             [defaultDate]="action === 'Create'? true : false"
```
Also lets use const for TableActions for `Create` instead of hard coding

Reply:
Changes are done